vpr/test/test_xy_routing.cpp

```diff
@@ -3,15 +3,13 @@
 
 #include "xy_routing.h"
 
```

Comment:
Probably good to put anonymous namespace back in. (Good to do for all these test files that had them .... Srivatsan did a nice job on that ).